src/features/courses/course-repository.ts

```diff
@@ -666,19 +678,28 @@ class CourseRepository {
                 logger.error('A single question was expected to update, but multiple update objects are returned.');
             }
 
-            if (updates[0] === 1 && !_.isEmpty(options.updates.courseQuestionAssessmentInfo)) {
+            if (updates[0] === 1 &&
+                !_.isEmpty(options.updates.courseQuestionAssessmentInfo) &&
```

Comment:
NIT: Order is different from above, I feel like I like nil check first, doesn't matter either way